main.py

```diff
@@ -1174,16 +1189,9 @@ def handle_link_down(self, interface):
         """Notify a link is down."""
         with self._links_lock:
             link = self._get_link_from_interface(interface)
-            if not link or not link.get_metadata("last_status_is_active"):
-                self.notify_topology_update()
-                return
-            link.deactivate()
-            metadata = {
-                "last_status_change": time.time(),
-                "last_status_is_active": False,
```

Comment:
Glad to see `last_status_is_active` being nuked. What we have here is easier to understand and follow. 

Let's also remove them from this statement here: https://github.com/kytos-ng/topology/blob/master/main.py#L162-L164. 

Finally, since this PR is removing these from metadata, let's also ship a script that will unset the DB documents accordingly, let's not leave unused stored properties anymore, especially since they're also exposed too, so can lead to confusion.

Reply:
Added in the script for retiring the metadata. It's a general script, but has a specific example for the given scenario.